packages/core/src/progress-circle/progress-circle.stories.ts

```diff
@@ -0,0 +1,199 @@
+        role="progressbar"
+        aria-valuenow="12"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        value="100"
+        role="progressbar"
+        aria-valuenow="100"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="info"
+        value="48"
+        role="progressbar"
+        aria-valuenow="48"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="success"
+        value="72"
+        role="progressbar"
+        aria-valuenow="72"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="warning"
+        value="36"
+        role="progressbar"
+        aria-valuenow="36"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="danger"
+        value="84"
+        role="progressbar"
+        aria-valuenow="84"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="unknown"
+        value="60"
+        role="progressbar"
+        aria-valuenow="60"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <div style="background: #313131">
+        <cds-progress-circle
+          size="xl"
+          inverse
+          value="24"
+          role="progressbar"
+          aria-valuenow="24"
+          aria-valuemin="0"
+          aria-valuemax="100"
+        ></cds-progress-circle>
+      </div>
+    </div>
+  `;
+};
+
+// this is kept here so we can verify the animation when we need to.
+// to test in screen reader, need to have a long interval.
+// note for future dev, throttle dynamic progress updates so as not to spam the screen reader.
+export const chaos = () => {
+  return html`
+    <span id="progress-announce" cds-layout="display:screen-reader-only" aria-live="polite"
+      >Circular progress example: 12% complete</span
+    >
+    <cds-progress-circle
+      value="12"
+      size="xl"
+      id="pureChaos"
+      role="progressbar"
+      aria-valuenow="12"
+      aria-valuemin="0"
+      aria-valuemax="100"
+    ></cds-progress-circle>
+    <br />
+    <button
+      @click=${() => {
+        setInterval(() => {
+          const el = document.getElementById('pureChaos');
+          const a11yEl = document.getElementById('progress-announce');
+          const newValue = Math.floor(Math.random() * (100 - 1)).toString();
+          el.setAttribute('value', newValue);
+          el.setAttribute('aria-valuenow', newValue);
+          a11yEl.innerText = `Circular progress example: ${newValue}% complete`;
+        }, 3200);
```

Comment:
Spamming the screen reader is a real risk with dynamically updating regions. This is something we should keep in mind when we advise product teams on improving screen reader experience.